cumulus/parachains/integration-tests/emulated/chains/parachains/bridges/bridge-hub-rococo/src/genesis.rs

```diff
@@ -22,6 +22,7 @@ use emulated_integration_tests_common::{
 };
 use parachains_common::Balance;
 
+pub const ASSETHUB_PARA_ID: u32 = 1000;
```

Comment:
Can we use `AssetHubRococo::para_id()` here?

Also perhaps the BridgeHub `PARA_ID` is also available in some constant provided by the bridges team

Reply:
It would require adding more dependencies. If this is out of date, the tests will fail. So I think it's simpler and protected against change.

Reply:
ok cool